phpbench.json

```diff
@@ -0,0 +1,4 @@
+{
+  "$schema":"./vendor/phpbench/phpbench/phpbench.schema.json",
+  "runner.bootstrap": "vendor/autoload.php"
+}
```

Comment:
Add an empty line at the end. You can configure PhpStorm to do that automatically on file save.